### ISSUE TITLE
Ability to do refunds

### ISSUE THREAD

**Issue by absalomedia** [flaviocopes/grav-plugin-shoppingcart#30]
Obviously as payments come in, there needs some capacity for the shopping cart to refund orders for customers on the payment gateways installed.

Most likely requires some new trigger states: onShoppingCartRefund & similar?

**Comment by flaviocopes** [comment]
Yes this will likely be supported by a plugin. Currently refunds must be done directly on the payment processor site.
